shaman/models/repos.py

```diff
@@ -52,16 +52,23 @@ def __repr__(self):
             return '<Repo detached>'
 
     def __json__(self):
+        from shaman.util import parse_distro_release
```

Comment:
circular import problems. ``shaman.util`` imports ``shaman.models``. putting this here was a quick fix.